### PR TITLE
Add Usages to Input Remoting

### PR DESCRIPTION
**Pull request by StayTalm**
Input Remoting wasn't sending usages on new device events.  This meant that the profiler wouldn't add usages to devices when connecting by default.

### CODE REVIEW

File: Packages/com.unity.inputsystem/CHANGELOG.md [hunk 1]
```diff
@@ -54,6 +54,7 @@ however, it has to be formatted properly to pass verification tests.
   * Regression introduced in 1.1-preview.2.
 - Fixed `Touch.activeTouches` having incorrect touch phases after calling `EnhancedTouch.Disable()` and then `EnhancedTouch.Enable()` ([case 1286865](https://issuetracker.unity3d.com/issues/new-input-system-began-moved-and-ended-touch-phases-are-not-reported-when-a-second-scene-is-loaded)).
 - Fixed compile errors related to XR/AR on console platforms.
+- Fixed issue with devices with usages not reporting them correctly in the remote profiler.
```
Comment (jimon) [inline Packages/com.unity.inputsystem/CHANGELOG.md]: It's a changelog for already released version :P

Comment (StayTalm) [reply Packages/com.unity.inputsystem/CHANGELOG.md]: ooops, did the same thing in the other PR too

**Comment by StayTalm** [comment]
Fixed changelog location, added a test, and ran the formatter

File: Packages/com.unity.inputsystem/CHANGELOG.md [hunk 2]
```diff
@@ -23,6 +23,7 @@ however, it has to be formatted properly to pass verification tests.
 - Fixed compile error on tvOS due to step counter support for iOS added in `1.1.0-preview.3`.
 - Fixed PS4- and PS3-specific `rightTriggerButton` and `leftTriggerButton` controls not being marked as synthetic and thus conflicting with `rightTrigger` and `leftTrigger` input ([case 1293734](https://issuetracker.unity3d.com/issues/input-system-when-binding-gamepad-controls-triggerbutton-gets-bound-instead-of-triggeraxis)).
   * This manifested itself, for example, when using interactive rebinding and seeing `rightTriggerButton` getting picked instead of the expected `rightTrigger` control.
+- Fixed issue with devices with usages not reporting them correctly in the remote profiler.
```
Comment (Rene-Damm) [inline Packages/com.unity.inputsystem/CHANGELOG.md]: Suggest `- Fixed changes to usages of devices in remote player not being reflected in Input Debugger.` or something like that.

**Review by Rene-Damm — APPROVED** [review]
Minor thing with the CHANGELOG. Otherwise LGTM.

File: Packages/com.unity.inputsystem/CHANGELOG.md (hunk 2 above)
Comment (StayTalm) [reply Packages/com.unity.inputsystem/CHANGELOG.md]: Swapped to that one

**Pull request merged**